openslides/agenda/static/templates/agenda/item-list.html

```diff
@@ -104,7 +104,7 @@ <h1 translate>Agenda</h1>
             </a>
           </li>
           <!-- CSV export -->
-          <li>
+          <li os-perms="motions.can_manage">
```

Comment:
We should not couple the agenda app to the motion app. Agenda is a dependency of motion, not vice versa.